test-app/tests/unit/active-model-adapter-test.js

```diff
@@ -235,7 +232,7 @@ module(serializer, function (hooks) {
     });
 
     test('using custom serialize keys function for transforming attributes and relationship keys', function (assert) {
-      let serializer = FactoryGuy.store.serializerFor('application');
+      let serializer = FactoryGuy.store.serializerFor('profile');
```

Comment:
Due to above changes in the test setup, making changes to application serializer isn't really going to apply for the profile serializer